chipsec/helper/linux/helper.py

```diff
@@ -204,6 +222,28 @@ def ioctl(self, nr, args, *mutate_flag):
 ###############################################################################################
 # Actual API functions to access HW resources
 ###############################################################################################
+    def memory_mapping(self, base, size):
+        """Returns the mmap region matching this area."""
```

Comment:
suggest replace "matching" with "that fully encompasses":
Returns the mmap region that fully encompasses this area, or None if no region matches.